lib/modules/manager/pixi/lockfile.ts

```diff
@@ -6,7 +6,7 @@
  *
  */
 const pixiGenerateLockFileVersion: Config[] = [
-  { lockVersion: 6, range: '>=0.39.2, <0.43.0' },
+  { lockVersion: 6, range: '>=0.40.0, <0.45.0a0' },
```

Comment:
```suggestion
  { lockVersion: 6, range: '>=0.40.0' },
```
Shouldn't this be open ended? 
If the lock file versions is incompatible the tool version extraction will simply fail, right? Which would the same situation as if the lockfile range here is not updated.

Reply:
We have upper boundary because we do not want to upgrade the version of lock file

Reply:
Pixi doesn't store the version of pixi when generating executable, so we choose a pixi version range based on lock file version, and store a mapping here

Reply:
https://github.com/renovatebot/renovate/pull/34646/files#diff-0b3f2b8e099fe5907ef0ab15fd1f415dd6aa379fdaaa2fa606f50325b97ebba3